web/unit_tests/test.sh

```diff
@@ -1,5 +1,13 @@
 #! /bin/bash
 
+WORKING_DIRECTORY=`pwd`
+
+# Include useful testing resource functions
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+KEYMAN_ROOT="$(dirname "$THIS_SCRIPT")/../.."
+. "$KEYMAN_ROOT/resources/build/build-utils.sh"
```

Comment:
Please use standard build script include (including comments):

```suggestion
## START STANDARD BUILD SCRIPT INCLUDE
# adjust relative paths as necessary
THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
## END STANDARD BUILD SCRIPT INCLUDE
```

Reply:
Right.  I implemented these sections before the comments on #2997 and such occurred and forgot to update the new instances contained here; good catch.  (Before that, KMW actually didn't have the standard build-script include on test.sh at all!)